src/cdk/explorer/tree/types.ts

```diff
@@ -60,5 +60,13 @@ export enum CfnResourceKeys {
  */
 export interface ConstructAttributes {
     readonly type: string
+    readonly props: ConstructProps
+}
+
+/**
+ * Represents a bag of properties
+ * Construct properties can be arrays, objects, strings, or boolean
```

Comment:
Should these be added instead of any?

Reply:
the `any` is still needed because those values are not represented in JSON unfortunately (i.e. a true (boolean) vs a true (string value) is not distinguishable

Reply:
kind of a limitation fo representing everything as a string